tests/foreman/ui/test_activationkey.py

```diff
@@ -211,378 +158,236 @@ def test_positive_create_activation_key_3(self, env_name):
+                description=description,
             )
+            self.assertIsNotNone(self.activationkey.wait_until_element(
+                common_locators['common_haserror']))
             self.assertIsNone(self.activationkey.search_key(name))
 
-    @skip_if_bug_open('bugzilla', 1139576)
     @data(*invalid_names_list())
-    def test_negative_create_activation_key_3(self, limit):
-        """@Test: Create Activation key with invalid Usage Limit
+    def test_negative_create_activation_key_with_invalid_limit(self, limit):
+        """@Test: Create Activation key with invalid Usage Limit. Both with too
+        long numbers and using letters.
 
         @Feature: Activation key - Negative Create
 
-        @Steps:
-        1. Create Activation key for all invalid Usage Limit in [2]
-        using valid Name, Description, Environment, Content View
-
         @Assert: Activation key is not created. Appropriate error shown.
 
-        @BZ: 1139576
-
         """
-        name = gen_string('alpha', 10)
+        name = gen_string('alpha')
         with Session(self.browser) as session:
             make_activationkey(
                 session,
-                org=self.org_name,
+                org=self.organization.name,
                 name=name,
                 env=ENVIRONMENT,
-                limit=limit
-            )
-            invalid = self.activationkey.wait_until_element(
-                locators['ak.invalid_limit']
+                limit=limit,
             )
-            self.assertIsNotNone(invalid)
+            self.assertIsNotNone(self.activationkey.wait_until_element(
+                locators['ak.invalid_limit']))
             self.assertIsNone(self.activationkey.search_key(name))
 
     @data(*valid_data_list())
-    def test_positive_delete_activation_key_1(self, name):
+    def test_positive_delete_activation_key_with_different_names(self, name):
         """@Test: Create Activation key and delete it for all variations of
         Activation key name
 
         @Feature: Activation key - Positive Delete
 
-        @Steps:
-        1. Create Activation key for all valid Activation Key names in [1]
-        using valid Description, Environment, Content View, Usage limit
-        2. Delete the Activation key
-
         @Assert: Activation key is deleted
 
         """
         with Session(self.browser) as session:
             make_activationkey(
                 session,
-                org=self.org_name,
+                org=self.organization.name,
                 name=name,
                 env=ENVIRONMENT,
-                description=gen_string('alpha', 16)
+                description=gen_string('utf8'),
             )
             self.assertIsNotNone(self.activationkey.search_key(name))
-            self.activationkey.delete(name, True)
-            self.assertIsNone(self.activationkey.search_key(name))
-
-    @data(*valid_data_list())
-    def test_positive_delete_activation_key_2(self, description):
```

Comment:
That scenario seems as pretty redundant and can be easily verified in scope of `test_positive_delete_activation_key_with_different_name`